dpnp/random/_random.pyx

```diff
@@ -44,7 +44,7 @@ from dpnp.dparray cimport dparray
 from dpnp.dpnp_utils cimport *
 
 
-ctypedef void(*fptr_mkl_rng_gaussian_1out_t)(void *, size_t)
+ctypedef void(*fptr_mkl_rng_gaussian_1out_t)(void *, double, double, size_t)
```

Comment:
It might be better to use some common type for 2 and 3 parameter (double, double here) to avoid all these types duplication.
Later, these parameters can be casted to _DataType inside kernel